lightningd/json.c

```diff
@@ -143,7 +143,7 @@ bool json_tok_short_channel_id(struct command *cmd, const char *name,
 
 	command_fail(cmd, JSONRPC2_INVALID_PARAMS,
 		     "'%s' should be a short channel id, not '%.*s'",
```

Comment:
```
lightning-cli check listchannels foo
{ "code" : -32602, "message" : "'short_channel_id' should be a short channel id, not '\"foo\"'" }
```
Since we are now properly escaping double quotes, then surrounding them with single quotes may not be necessary anymore (as in `'\"foo\"'` above).  And then maybe change `'short_channel_id'` to `\"short_channel_id\"` also?

Of course this would affect many places in the code.

Reply:
Generally I find ' quotes clearer for errors like this, TBH, so I think we leave it.  I like the quotes in case spaces sneak in, too.

Reply:
👍